### PR TITLE
lttp: add EnemyPool option

### PR DESCRIPTION
**Pull request by alwaysintreble**
Acts as an option list, which are then passed into enemizer which places them wherever the enemizer logic deems those enemies valid. Needs testing

### CODE REVIEW

**Comment by Berserker66** [comment]
This is stated somewhat confusingly in the description. Does this filter the available enemies if shuffle is on?

**Comment by Ijwu** [comment]
Note: requires updated enemizer

**Comment by alwaysintreble** [comment]
> This is stated somewhat confusingly in the description. Does this filter the available enemies if shuffle is on?

This requires enemy shuffle to be enabled, it then forces the listed enemies in all "valid" locations and then enemizer places random enemies in all remaining locations that the supplied list of enemies are not valid in.

File: worlds/alttp/Options.py [hunk 1]
```diff
@@ -198,6 +198,27 @@ class EnemyShuffle(Toggle):
     display_name = "Enemy Shuffle"
 
 
+class PlandoEnemies(OptionList):
+    """Define a list of enemies to be placed where possible. Where they cannot be placed will be randomized enemies.
+    Only enabled when enemy shuffle is enabled. For valid enemies see
+    https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/alttp/Options.py#L201"""
```
Comment (Alchav) [inline worlds/alttp/Options.py:204]: If you set the default value for the option to a list of all enemies, then the generated yaml template will come with a list of all of them as the default

**Comment by Berserker66** [comment]
Now you're describing swaps but it's a list?

**Pull request closed without merging**